x/dex/keeper/grpc_query_estimate_multi_hop_swap.go

```diff
@@ -8,14 +8,13 @@ import (
 	"github.com/neutron-org/neutron/v4/x/dex/types"
 )
 
-// TODO: This doesn't run ValidateBasic() checks.
 func (k Keeper) EstimateMultiHopSwap(
 	goCtx context.Context,
 	req *types.QueryEstimateMultiHopSwapRequest,
 ) (*types.QueryEstimateMultiHopSwapResponse, error) {
 	msg := types.MsgMultiHopSwap{
-		Creator:        req.Creator,
-		Receiver:       req.Receiver,
+		Creator:        "neutron1dft8nwxzr0u27wvr2cknpermjkreqvp9fdy0uz",
+		Receiver:       "neutron1dft8nwxzr0u27wvr2cknpermjkreqvp9fdy0uz",
```

Comment:
What is this address?

Reply:
Probably need to create another message `QueryMultiHopSwap` with it's own validate and reusing validate logic somehow if possible.
Otherwise kinda ugly